### ISSUE TITLE
2️⃣ Fix Potential Training Behavior

### ISSUE THREAD

**Issue by Microcosmos22** [Microcosmos22/Mask-R-CNN#2]
You currently set:

model.train()  # in validation

That keeps dropout + batchnorm updating. Validation should be:

model.eval()

Otherwise val loss can behave misleadingly.
